wdl-lint/src/lib.rs

```diff
@@ -80,6 +80,17 @@ pub trait Rule: Visitor<State = Diagnostics> {
     ///
     /// If `None` is returned, all nodes are exceptable.
     fn exceptable_nodes(&self) -> Option<&'static [SyntaxKind]>;
+
+    /// Gets the ID of rules that are related to this rule.
+    ///
+    /// This can be used by tools (like `sprocket explain`) to suggest other
+    /// relevant rules to the user based on potential logical connections or
+    /// common co-occurrences of issues.
+    ///
+    /// By default, this returns an empty list.
+    fn related_rules(&self) -> Vec<&'static str> {
```

Comment:
```suggestion
    fn related_rules(&self) -> &[&str] {
```
I think returning a slice should be more performant (although in practice I don't think it will make a difference). This will avoid a new allocation for each invocation of `related_rules()`